protocol/indexer/indexer_manager/event_manager.go

```diff
@@ -64,6 +74,20 @@ func (i *indexerEventManagerImpl) SendOnchainData(block *IndexerTendermintBlock)
 	}
 }
 
+// AddOnchainStreamEvent adds a onchain stream event to the context's transient store.
+// TODO(CT-939): Consolidate FNS with indexer events; share storage and event emission logic.
+func (i *indexerEventManagerImpl) AddOnchainStreamEvent(
+	ctx sdk.Context,
+	dataBytes []byte,
+) {
+	event := IndexerTendermintEventWrapper{
+		Event: &IndexerTendermintEvent{
+			DataBytes: dataBytes,
+		},
+	}
+	addEvent(ctx, event, i.onchainStreamTransientStoreKey)
```

Comment:
The code here https://github.com/dydxprotocol/v4-chain/blob/e09241b99990611e89eca40737ff1df1ab2416f2/protocol/indexer/indexer_manager/events.go#L118
increments indexer `IndexerEventsCountKey`. We shouldn't do that for transient events right?